tests/testthat/test-colcheck.R

```diff
@@ -95,7 +95,10 @@ test_that("non-standard roles during bake/predict", {
 
   # This should require 'date' to predict.
   # The error comes from hardhat, so we don't snapshot it because we don't own it.
```

Comment:
I think this is a good principle to rely on, i.e. I would suggest not doing the change below and similar. If you want a more thorough test than "just make sure there is _a_ error" you could see if it's a classed error and check for that error class here.

Reply:
I'm unresolving this because it seems like you might have hit "resolved" by accident?